test/e2e/specs/templates.test.js

```diff
@@ -1,25 +1,77 @@
 /**
  * Internal dependencies
  */
-import { clickOnMoreMenuItem, newPost } from '../support/utils';
+import {
+	clickOnMoreMenuItem,
+	newPost,
+	visitAdmin,
+	getEditedPostContent,
+	clickBlockAppender,
+	saveDraft,
+} from '../support/utils';
 import { activatePlugin, deactivatePlugin } from '../support/plugins';
 
-describe( 'Using a CPT with a predefined template', () => {
-	beforeAll( async () => {
-		await activatePlugin( 'gutenberg-test-plugin-templates' );
-		await newPost( { postType: 'book' } );
-	} );
+describe( 'templates', () => {
+	describe( 'Using a CPT with a predefined template', () => {
```

Comment:
None of these existing "Using a CPT" test cases have been changed. They're simply grouped / nested within a new "templates" `describe` block.